packages/clerk-js/package.json

```diff
@@ -36,7 +36,8 @@
     "lint": "eslint .",
     "start": "echo \"Noop\"",
     "test": "jest",
-    "test:coverage": "jest --collectCoverage && open coverage/lcov-report/index.html"
+    "test:coverage": "jest --collectCoverage && open coverage/lcov-report/index.html",
+    "watch": "webpack --config webpack.config.js --env production --watch"
```

Comment:
Added this command to pair with the playground file I added here while developing the feature. When running this in the terminal, it will rebuild changed files more quickly than running a full build after every save.

Reply:
Makes sense to me. Just curious, does HMR work correctly when using `playground.html`? I *think* HMR won't work unless the file is served under `https` as HMR relies on webhooks.